demo/src/main/java/net/minestom/demo/PlayerInit.java

```diff
@@ -216,8 +221,8 @@ public static void init() {
                     .append(Component.text("TICK TIME: " + MathUtils.round(tickMonitor.getTickTime(), 2) + "ms"))
                     .append(Component.newline())
                     .append(Component.text("ACQ TIME: " + MathUtils.round(tickMonitor.getAcquisitionTime(), 2) + "ms"));
-            final Component footer = benchmarkManager.getCpuMonitoringMessage();
-            Audiences.players().sendPlayerListHeaderAndFooter(header, footer);
-        }).repeat(10, TimeUnit.SERVER_TICK); //.schedule();
+            final Component footer = BenchmarkManager.getCpuMonitoringMessage(benchmarkManager);
+            serverFacade.getAudienceManager().players().sendPlayerListHeaderAndFooter(header, footer);
+        }).repeat(10, TimeUnit.getServerTick(serverFacade.getServerSettings())); //.schedule();
```

Comment:
IMO another method should be added to take a SERVER_TICKS constant which the scheduler can then get from the server rather than it being a TimeUnit and requiring a reference to the server here.

Reply:
SERVER_TICKS is no longer a constant, as it is a mutable parameter that may differ between server instances

Reply:
Yes I mean there should be a constant to represent server ticks and converted when encountered

Reply:
help needed